features/admin.extensions.v1/components/tenants/api/tenants.ts

```diff
@@ -30,6 +30,11 @@ export const getDomainQueryParam = (): string => {
     return `?domain=${ tenantDomain }`;
 };
 
+export const getLimitQueryParam = (): string => {
+
+    return "&limit=30";
```

Comment:
Won't we have a similar problem when someone says their 31st tenant is not in the tenant dropdown?

I understand this is kinda rare case, but 15 was also (probably) considered as a rare case back then.

Wouldn't a paginated dropdown list be the ideal solution for this problem?

Reply:
As I checked with the issue discussions, this is a temporary measure and the paginated dropdown is planned as a separate improvement. Hence +1.